### DISCUSSION TITLE
C# 12 - Interceptors

### DISCUSSION THREAD

**Discussion by The-Futurist** [dotnet/csharplang#7601] [General] [answered]
In the docs this feature is singled out:

![image](https://github.com/dotnet/csharplang/assets/12262952/c61093df-b148-4fcd-9c6c-1277fc8bc387)

So at this quite late stage, is there more risk of interceptors being omitted than the other C# 12 features? Since its described as an "experimental feature" does that make it any different to the other expected C# 12 features?

**Comment by CyrusNajmabadi** [comment, marked as answer]
Interceptors is shipping as preview in C#12.  Future plans are completely up in the air past that point.

**Reply by The-Futurist** [reply]
Thanks.
